core-bundle/src/Resources/contao/templates/_new/content_element/table.html.twig

```diff
@@ -0,0 +1,79 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{# todo: remove once the table sorting script was adjusted or inlined #}
+{% set attributes = attrs().addClass('ce_table') %}
+
+{% block content %}
+    {% set table_attributes = attrs(table_attributes|default)
+        .set('id', 'table_' ~ data.id, sorting)
+        .addClass('sortable', sorting)
+        .set('data-sort-default', sorting.column|default ~ '|' ~ sorting.order|default, sorting)
+    %}
```

Comment:
```suggestion
    {% set table_attributes = attrs(table_attributes|default) %}
    {% if sorting %}
        {% table_attributes
            .set('id', 'table_' ~ data.id)
            .addClass('sortable')
            .set('data-sort-default', sorting.column|default ~ '|' ~ sorting.order|default)
        %}
    {% endif %}
```
Would something like this be more readable?

Reply:
Had this before, but then changed it to the inline version. :smile: 

But already convinced. I used the `{% do %}` tag in 69828a6251c752e1d073c458ec3944c7d0fbb7b2.